<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>946</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>pyBlink</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalImagesLayout" stretch="0,0">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
          <item alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QGraphicsView" name="detailsView">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>8</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Target</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Filter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RotatorAngle</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PierSide</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status Others</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Startrails</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalRadioButtonsLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_all">
          <property name="text">
           <string>Show All Subs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_notdiscarded">
          <property name="text">
           <string>Show not Discarded and New Subs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_discarded">
          <property name="text">
           <string>Show Discarded Subs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_discardedbyothers">
          <property name="text">
           <string>Show Discarded by Others Subs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_startrails">
          <property name="text">
           <string>Subs with Startrails</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_nostartrails">
          <property name="text">
           <string>Subs without Startrails</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionS3Open"/>
   <addaction name="actionSync"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open from local filesystem</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSync">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
   </property>
   <property name="text">
    <string>Sync with S3 Share</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionS3Open">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::NetworkWired"/>
   </property>
   <property name="text">
    <string>Open from S3 Share</string>
   </property>
   <property name="toolTip">
    <string>Open from S3 Share</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
